pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -236,13 +246,25 @@ let
         for src in ./jaxlib/*.{cc,h}; do
           sed -i 's@include/pybind11@pybind11@g' $src
         done
-        sed -i 's@-lprotobuf@-l:libprotobuf.a@' ../output/external/org_tensorflow/third_party/systemlibs/protobuf.BUILD
-        sed -i 's@-lprotoc@-l:libprotoc.a@' ../output/external/org_tensorflow/third_party/systemlibs/protobuf.BUILD
         substituteInPlace ../output/external/org_tensorflow/tensorflow/compiler/xla/python/pprof_profile_builder.cc \
           --replace "status.message()" "std::string{status.message()}"
+        substituteInPlace ../output/external/rules_cc/cc/private/toolchain/osx_cc_wrapper.sh.tpl \
+          --replace "/usr/bin/install_name_tool" "${cctools}/bin/install_name_tool"
+        substituteInPlace ../output/external/rules_cc/cc/private/toolchain/unix_cc_configure.bzl \
+          --replace "/usr/bin/libtool" "${cctools}/bin/libtool"
```

Comment:
This broke `jaxlib` on non-Darwin because `cctools` is Darwin-specific.

https://github.com/NixOS/nixpkgs/blob/4dc0fce489d180acb37a52f6569068175b9a7954/pkgs/os-specific/darwin/cctools/port.nix#L100

Reply:
Unfortunate, it wasn't evaluating on ofborg so I couldn't check, have you already fixed it? I'll put up a fix.

Reply:
https://github.com/NixOS/nixpkgs/pull/187536